### ISSUE TITLE
Input to Output mapping

### ISSUE THREAD

**Issue by iliaishacked** [citp/BlockSci#91]
Hihi!

Is there any public way to find an Output from a given Input? I can see that there is a way to return the Transaction that created the Input, but I cant see any way to actually tell what the output index is in the Transaction. 

I can see in the example that there has been some function named "input.getSpentOutput();"
https://github.com/citp/BlockSci/blob/master/src/example/example.cpp#L224

Is there any chance you could return this functionality or show the source code for it?

Thanks

**Comment by hkalodner** [comment]
Hi @iliaishacked,

This functionality was included when inputs were just pointers to output data. We moved a a data model where input data was inlined and include only a reference to the transaction they spent rather than the specific output of that transaction so this is not available anymore. At the time we couldn't think of any need for knowing the specific output, but it appears that it would actually be useful.

I've added this to the todo list for the next update of the parser. It should be a relatively simple and performant addition.
